### PR TITLE
chore: release v4.7.1

### PR DESCRIPTION
**Pull request by Haroenv**
## Release Summary
- Version change: `v4.7.0` → `v4.8.0`
- Merge: `releases/v4.8.0` → `master`
- [Compare the changes between the versions](https://github.com/algolia/instantsearch.js/compare/v4.7.0...releases/v4.8.0)
> :warning: When merging this pull request, you need to **_"Squash and merge"_** and make sure the title starts with `chore: release v4.8.0`.
> After that, a commit `chore: release v4.8.0` will be added and `shipjs trigger` will be able to trigger the release based on the commit.
> Fore more information, please refer to the mergeStrategy section of the [guide](https://community.algolia.com/shipjs/guide/useful-config.html#mergestrategy).
> Squash and merge

---
This is going to be published by the following command:
```
yarn publish --no-git-tag-version --non-interactive --tag latest
```
---
_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+* **dev:** don't show a warning for connectRange ([#4440](https://github.com/algolia/instantsearch.js/issues/4440)) ([eb8c8b3](https://github.com/algolia/instantsearch.js/commit/eb8c8b3494cb66dbef1d03e7d74374dc49059345)), closes [#4437](https://github.com/algolia/instantsearch.js/issues/4437)
+* **infiniteHits:** work with controlled mode ([#4435](https://github.com/algolia/instantsearch.js/issues/4435)) ([68b20f4](https://github.com/algolia/instantsearch.js/commit/68b20f487fcd54fd7dec11b4c494b6aa94a18516))
+* **ts:** correct dummy v4 client ([#4459](https://github.com/algolia/instantsearch.js/issues/4459)) ([ca0c394](https://github.com/algolia/instantsearch.js/commit/ca0c3946608bb8ec5dcf5378d8d382d809a4d86f))
+* **ts:** jsDoc comments which conform to Connector definition ([#4458](https://github.com/algolia/instantsearch.js/issues/4458)) ([5209bdb](https://github.com/algolia/instantsearch.js/commit/5209bdb9189e7cbbf9514b62fde55f923b2b3273))
```
Comment (Haroenv) [inline CHANGELOG.md:11]:
```suggestion
* **typescript:** correct dummy v4 client ([#4459](https://github.com/algolia/instantsearch.js/issues/4459)) ([ca0c394](https://github.com/algolia/instantsearch.js/commit/ca0c3946608bb8ec5dcf5378d8d382d809a4d86f))
* **typescript:** jsDoc comments which conform to Connector definition ([#4458](https://github.com/algolia/instantsearch.js/issues/4458)) ([5209bdb](https://github.com/algolia/instantsearch.js/commit/5209bdb9189e7cbbf9514b62fde55f923b2b3273))
```

**Comment by francoischalifour** [comment]
Why isn't it version `4.7.1` if there are only bug fixes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
+# [4.8.0](https://github.com/algolia/instantsearch.js/compare/v4.7.0...v4.8.0) (2020-08-19)
```
Comment (Haroenv) [inline CHANGELOG.md:1]:
```suggestion
# [4.7.1](https://github.com/algolia/instantsearch.js/compare/v4.7.0...v4.7.1) (2020-08-19)
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "instantsearch.js",
-  "version": "4.7.0",
+  "version": "4.8.0",
```
Comment (Haroenv) [inline package.json:3]:
```suggestion
  "version": "4.7.1",
```

File: src/lib/version.ts [hunk 4]
```diff
@@ -1 +1 @@
-export default '4.7.0';
+export default '4.8.0';
```
Comment (Haroenv) [inline src/lib/version.ts:1]:
```suggestion
export default '4.7.1';
```

**Review by Haroenv — COMMENTED** [review]
change to patch

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,18 @@
+# [4.7.1](https://github.com/algolia/instantsearch.js/compare/v4.7.0...v4.7.1) (2020-08-19)
+
+
+### Bug Fixes
+
+* **configureRelatedItems:** support nested attributes ([#4480](https://github.com/algolia/instantsearch.js/issues/4480)) ([2266004](https://github.com/algolia/instantsearch.js/commit/2266004f274138b45640f000a5da8aa14e419e6c))
+* **connectToggleRefinement:** fix onFacetValue/offFacetValue on render when using arrays for on/off ([#4449](https://github.com/algolia/instantsearch.js/issues/4449)) ([fd3e83f](https://github.com/algolia/instantsearch.js/commit/fd3e83f2cf2e5b44b7d29eb4c67526e55c18d708))
+* **dev:** don't show a warning for connectRange ([#4440](https://github.com/algolia/instantsearch.js/issues/4440)) ([eb8c8b3](https://github.com/algolia/instantsearch.js/commit/eb8c8b3494cb66dbef1d03e7d74374dc49059345)), closes [#4437](https://github.com/algolia/instantsearch.js/issues/4437)
```
Comment (francoischalifour) [inline CHANGELOG.md:8]:
```suggestion
* **index:** don't show a development warning for inconsistent UI state in `connectRange` ([#4440](https://github.com/algolia/instantsearch.js/issues/4440)) ([eb8c8b3](https://github.com/algolia/instantsearch.js/commit/eb8c8b3494cb66dbef1d03e7d74374dc49059345)), closes [#4437](https://github.com/algolia/instantsearch.js/issues/4437)
```

**Review by francoischalifour — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (Haroenv) [reply CHANGELOG.md:8]: will merge after the tests have run so I don't have to wait for them again to see if e2e failure is consistent

**Pull request merged**